test/checks/language/valid-lang.js

```diff
@@ -43,13 +43,12 @@ describe('valid-lang', function () {
 			assert.deepEqual(checkContext._data, ['lang="en-FOO"']);
 		});
 
-		it('should return true (and not throw) when given no options', function () {
+		it('should return false (and not throw) when given no options', function () {
```

Comment:
It's related to the `none` in the check. It's a bit backwards. Previously, there were `options` or `[]`, but now we've got `(options ? options : axe.commons.utils.validLangs)` - so either `null` or some, but never empty.

This is my best explanation. It's kind of hard to wrap my head around the tests that are true, that return false :)